.github/workflows/opendownstream-pr.yml

```diff
@@ -105,7 +105,11 @@ jobs:
 
           BRANCH_NAME="sync/container-libs-${{ github.event.pull_request.number }}"
           PR_TITLE="Sync: ${SELF_REPO_PR_TITLE}"
-          PR_BODY="This PR automatically vendors changes from [container-libs#${SELF_REPO_PR_NUMBER}](${SELF_REPO_PR_URL}).\n This PR is intended solely for testing purposes and should not be merged. Ideally, upstream maintainers should review any failing tests here and close this PR once the upstream PR is merged. Maintainers of this repository do not need to take any action, but may review the changes and notify upstream maintainers of any breaking issues if they wish. \n\nThis PR is generated from a github actions workflow, please report any issues on https://github.com/containers/container-libs"
+          PR_BODY="This PR automatically vendors changes from [container-libs#${SELF_REPO_PR_NUMBER}](${SELF_REPO_PR_URL}).
+
+This PR is intended solely for testing purposes and should not be merged. Ideally, upstream maintainers should review any failing tests here and close this PR once the upstream PR is merged. Maintainers of this repository do not need to take any action, but may review the changes and notify upstream maintainers of any breaking issues if they wish.
+
+This PR is generated from a github actions workflow, please report any issues on https://github.com/containers/container-libs"
```

Comment:
that does not look like a valid yaml file, you must maintain the yaml indentation